lib/components/PrizeCard.jsx

```diff
@@ -102,28 +102,27 @@ const PrizeSection = (props) => {
 
   return (
     <>
-      {' '}
-      <CardTitle className='text-center'>Current Prize</CardTitle>
       <Prizes />
+      <CardTitle className='text-center mb-8' s>
+        Current Prize
+      </CardTitle>
     </>
   )
 }
 
 const Prizes = (props) => {
   const { awards, loading } = useAwardsList()
 
-  if (awards.length === 1) {
+  const awardsWithBalances = awards.filter((token) => token.formattedBalance != 0)
```

Comment:
I've gotten into the habit of only ever using `===` and `!==` in JS. Maybe this kind of equality is what you're looking for? This may be correct but wanted to check with you.

Also are `formatted*` vars not typically strings instead of ints/floats?

Reply:
It was intentional, the formatted balance was a string "0.000", but I can just tack on the big number balance and check that to be a little more precise. Using `!=` and `==` do feel dirty.